packages/preset-mini/src/preflights.ts

```diff
@@ -8,7 +8,8 @@ export const preflights: Preflight[] = [
     getCSS(ctx: PreflightContext<Theme>) {
       if (ctx.theme.preflightBase) {
         const css = entriesToCss(Object.entries(ctx.theme.preflightBase))
-        return `*,::before,::after{${css}}::backdrop{${css}}`
+        const root = ctx.theme.preflightRoot ?? ''
```

Comment:
I would prefer to make the entire selector customizable: 

```suggestion
        const root = ctx.theme.preflightRoot ?? ':root,:root::before,:root::after'
```

Reply:
Updated to arrayable since there's separated `::backdrop`